src/features/events/utils/getStatusDotLabel.tsx

```diff
@@ -0,0 +1,41 @@
+import { EventState } from '../models/EventDataModel';
+import messageIds from '../l10n/messageIds';
+import { Msg } from 'core/i18n';
+import { STATUS_COLORS } from 'features/campaigns/components/ActivityList/items/ActivityListItem';
+
+export default function getStatusDotLabel({
+  color,
+  state,
+}: {
+  color?: STATUS_COLORS;
+  state?: EventState;
```

Comment:
I don't love this, but I understand the need with the current mess that is status management. We will have to return to this later and clean up, maybe!

Reply:
Yep, it's a mess definitely 🤣